app/marketing/tests/management/commands/test_sync_keywords.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""Handle marketing commands related tests.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import date, datetime, timedelta
+from unittest.mock import patch
+
+from django.test import TestCase
+from django.utils import timezone
+
+from dashboard.models import Bounty, BountyFulfillment, Profile, Tip
+from marketing.management.commands.sync_keywords import Command
+from marketing.models import Keyword
+
+
+class TestSyncKeywords(TestCase):
+    """Define tests for sync keywords."""
+
+    def setUp(test):
```

Comment:
Can you switch this to `def setUp(self):` ?

Reply:
sure 👍